app/src/main/java/com/sunlightlabs/android/congress/fragments/BillListFragment.java

```diff
@@ -230,7 +232,7 @@ else if (!hasA && hasB)
 		}
 		
 		// only re-enable the pagination if we got a full page back
-		if (bills.size() == PER_PAGE)
+		if (bills.size() >= PER_PAGE)
```

Comment:
Made scroll logic slightly more flexible, in case we have to do client-side merging and return 40 results per-page for some requests and not 20.